rosdep/base.yaml

```diff
@@ -1656,13 +1656,22 @@ libgps:
   ubuntu: [libgps-dev]
 libgsl:
   arch: [gsl]
-  debian: [libgsl0-dev]
+  debian:
+    wheezy: [libgsl0-dev]
+    jessie: [libgsl0-dev]
+    stretch: [libgsl-dev]
+    sid: [libgsl-dev]
   fedora: [gsl]
   gentoo:
     portage:
       packages: [sci-libs/gsl]
   macports: [gsl]
-  ubuntu: [libgsl0-dev]
+  ubuntu:
+    precise: [libgsl0-dev]
```

Comment:
Although the other platforms are EOL, please fill in the intermediate platforms since there are people still using them with ROS starting at least from Saucy since that's Indigo.